samples/extensions/logic_op_dynamic_state/README.md

```diff
@@ -0,0 +1,166 @@
+
+```C++
+std::vector<VkDynamicState> dynamic_state_enables = {
+	VK_DYNAMIC_STATE_VIEWPORT,
+	VK_DYNAMIC_STATE_SCISSOR,
+	VK_DYNAMIC_STATE_LOGIC_OP_EXT,
+};
+VkPipelineDynamicStateCreateInfo dynamic_state =
+	vkb::initializers::pipeline_dynamic_state_create_info(
+	    dynamic_state_enables.data(),
+	    static_cast<uint32_t>(dynamic_state_enables.size()),
+	    0);
+```
+
+Both structures: `color_blend_state` and `dynamic_state` are used to define [`VkGraphicsPipelineCreateInfo`](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VkGraphicsPipelineCreateInfo.html) needed to create the graphics pipeline.
+
+```C++
+VkGraphicsPipelineCreateInfo graphics_create{VK_STRUCTURE_TYPE_GRAPHICS_PIPELINE_CREATE_INFO};
+graphics_create.pColorBlendState    = &color_blend_state;
+graphics_create.pDynamicState       = &dynamic_state;
+...
+
+VK_CHECK(vkCreateGraphicsPipelines(get_device().get_handle(), pipeline_cache, 1, &graphics_create, VK_NULL_HANDLE, &pipeline.baseline));
+```
+
+Logical operations are not enabled in the background pipeline.
+
+```C++
+std::vector<VkDynamicState> dynamic_state_enables_background = {
+    VK_DYNAMIC_STATE_VIEWPORT,
+    VK_DYNAMIC_STATE_SCISSOR,
+};
+
+/* Disable logic operations in background pipeline */
+color_blend_state.logicOpEnable = VK_FALSE;
+```
+
+## Logical operation dynamic setting in command buffer creation
+
+Calling [`vkCmdSetLogicOpEXT`](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkCmdSetLogicOpEXT.html) allows to set the logic operation dynamically. In the sample it is called before drawing the model in the `build_command_buffers` method using a value provided by the GUI. Available logical operations are defined by the `VkLogicOp` enumeration (check the Vulkan&reg; specification for details of [the logical operations](https://registry.khronos.org/vulkan/specs/1.3-extensions/html/vkspec.html#VkLogicOp)).
+The background model is drawn before the cube model (with logical operations enabled) to observe blending results on the cube model.
+
+```C++
+VK_CHECK(vkBeginCommandBuffer(draw_cmd_buffer, &command_begin));
+...
+
+/* Drawing background */
+draw_model(background_model, draw_cmd_buffer);
+...
+
+/* Set logic operation chosen in GUI for the cube model */
+vkCmdSetLogicOpEXT(draw_cmd_buffer, static_cast<VkLogicOp>(gui_settings.selected_operation));
+
+/* Draw model */
+draw_created_model(draw_cmd_buffer);
+...
+
+VK_CHECK(vkEndCommandBuffer(draw_cmd_buffer));
+```
+
+## Enabling the required extensions
+
+The extended dynamic state 2 API requires Vulkan&reg; 1.0 and the appropriate headers / SDK is required. This extension has been [partially](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_EXT_extended_dynamic_state2.html#_promotion_to_vulkan_1_3) promoted to Vulkan&reg; 1.3.
```

Comment:
I don't think we need to use the ® in readmes. I'd remove it for better readability.

Reply:
Fixed, thanks.